examples/tv-casting-app/tv-casting-common/core/CastingApp.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "CastingApp.h"
+
+#include <app/clusters/bindings/BindingManager.h>
+#include <app/server/Server.h>
+#include <credentials/DeviceAttestationCredsProvider.h>
+#include <credentials/attestation_verifier/DeviceAttestationVerifier.h>
+
+namespace matter {
+namespace casting {
+namespace app {
+
+using namespace matter::casting::support;
+
+CastingApp * CastingApp::castingApp_ = nullptr;
```

Comment:
nit, should it be `_castingApp` instead? i think i've seen that convention in other code

Reply:
Done.